modules/oadp-usecase-kopia-override-algorithms.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/overriding-kopia-algorithms.adoc          
+
+:_mod-docs-content-type: PROCEDURE
+[id="oadp-usecase-kopia-override-algorithms.adoc_{context}"]
+= Use case for overriding Kopia hashing, encryption, and splitter algorithms
+
+This use case walks you through the steps for taking a backup with Kopia environment variables for hashing, encryption, and splitter. You store the backup in an AWS S3 bucket. You then verify the environment variables by connecting to the Kopia repository.
+
+.Prerequisites
+
+* You have installed the {oadp-short} Operator.
+* You have an AWS S3 bucket configured as the backup storage location.
+* You have created the secret by using the credentials provided by the cloud provider.
+* You have installed the Kopia client.
+* You have an application with persistent volumes running in a separate namespace.
+
+.Procedure
+
+. Configure the DPA as shown in the following example:
++
+[source,yaml]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+name: <dpa_name> # <1>
+namespace: openshift-adp
+spec:
+backupLocations:
+- name: aws
+  velero:
+    config:
+      profile: default
+      region: <region_name> # <2>
+    credential:
+      key: cloud
+      name: cloud-credentials # <3>
+    default: true
+    objectStorage:
+      bucket: <bucket_name> # <4>
+      prefix: velero
+    provider: aws
+configuration:
+  nodeAgent:
+    enable: true
+    uploaderType: kopia
+  velero:
+    defaultPlugins:
+    - openshift
+    - aws
+    - csi # <5>
+    defaultSnapshotMoveData: true
+    podConfig:
+      env:
+        - name: KOPIA_HASHING_ALGORITHM
+          value: BLAKE3-256 # <6>
+        - name: KOPIA_ENCRYPTION_ALGORITHM
+          value: CHACHA20-POLY1305-HMAC-SHA256 # <7>
+        - name: KOPIA_SPLITTER_ALGORITHM
+          value: DYNAMIC-8M-RABINKARP # <8>
```

Comment:
```suggestion
          value: DYNAMIC-8M-RABINKARP # <8>
# ...
```

Reply:
The yaml is fully complete. I have myself tested the yaml on a QE cluster. Therefore, not including the '...'. Thanks.